test/requirements.txt

```diff
@@ -40,7 +40,7 @@ filelock==3.7.1
     # via
     #   huggingface-hub
     #   transformers
-gradio_pdf==0.0.3
+gradio_pdf==0.0.5
```

Comment:
Updated the package to use the set_documentation_group api

Reply:
Cool let's keep this pinned from now on so that we can test backwards compatibility